pom.xml

```
@@ -35,6 +27,161 @@
     </license>
   </licenses>
 
+  <scm>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <tag>${scmTag}</tag>
+    <url>https://github.com/${gitHubRepo}</url>
+  </scm>
+
+  <properties>
+    <revision>6.20</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jetty.version>10.0.20</jetty.version>
+    <slf4j.version>2.0.13</slf4j.version>
+    <gitHubRepo>jenkinsci/winstone</gitHubRepo>
+    <spotless.check.skip>true</spotless.check.skip>
```

Why are we leaving Spotless checking skipped in a PR that enables Spotless?

```suggestion
    <spotless.check.skip>false</spotless.check.skip>
```